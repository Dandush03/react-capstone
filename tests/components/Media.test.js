import React from 'react';
import renderer from 'react-test-renderer';
import Media from '../../src/components/Media';

describe('Media', () => {
  test('Render Elements', () => {
    const component = renderer.create(<Media />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
