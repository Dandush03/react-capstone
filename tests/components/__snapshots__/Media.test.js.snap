// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Media Render Elements 1`] = `
<div
  className="media-icons"
>
  <a
    className="linkedin-hero"
    href="https://www.linkedin.com/in/daniel-laloush"
  >
    <svg
      aria-hidden="true"
      className="linkedin"
      data-icon="linkedin"
      data-prefix="fab"
      focusable="false"
      role="img"
      viewBox="0 0 448 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
        fill="currentColor"
      />
    </svg>
  </a>
  <a
    className="hero-page"
    href="https://dlaloush.me/"
  >
    <svg
      aria-hidden="true"
      className="hero"
      data-icon="paper-plane"
      data-prefix="fas"
      focusable="false"
      role="img"
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M476 3.2L12.5 270.6c-18.1 10.4-15.8 35.6 2.2 43.2L121 358.4l287.3-253.2c5.5-4.9 13.3 2.6 8.6 8.3L176 407v80.5c0 23.6 28.5 32.9 42.5 15.8L282 426l124.6 52.2c14.2 6 30.4-2.9 33-18.2l72-432C515 7.8 493.3-6.8 476 3.2z"
        fill="currentColor"
      />
    </svg>
  </a>
</div>
`;
